modules/imaging_browser/jsx/imagingBrowserIndex.js

```diff
@@ -0,0 +1,199 @@
+                cellLinks.push(<a key="all" href={loris.BaseURL +
+                '/imaging_browser/viewSession/?sessionID=' +
+                row.SessionID +
+                '&backURL=/imaging_browser/'}>
+                    all types
+                </a>);
+                result = (<td>{cellLinks}</td>);
+                break;
+        }
+
+        return result;
+    }
+
+    render() {
+        // If error occurs, return a message.
+        // XXX: Replace this with a UI component for 500 errors.
+        if (this.state.error) {
+            return <h3>An error occured while loading the page.</h3>;
+        }
+
+        // Waiting for async data to load
+        if (!this.state.isLoaded) {
+            return <Loader/>;
+        }
+
+        /**
+         * Currently, the order of these fields MUST match the order of the
+         * queried columns in _setupVariables() in imaging_browser.class.inc
+         */
+        const options = this.state.data.fieldOptions;
+        const configLabels = options.configLabels;
+        const fields = [
+            {label: 'Site', show: true, filter: {
+                name: 'site',
+                type: 'select',
+                options: options.sites,
+            }},
+            {label: 'PSCID', show: true, filter: {
+                name: 'PSCID',
+                type: 'text',
+            }},
+            {label: 'DCCID', show: true, filter: {
+                    name: 'DCCID',
+                    type: 'text',
+            }},
+            {label: 'Project', show: true, filter: {
+                name: 'project',
+                type: 'select',
+                options: options.projects,
+            }},
+            {label: 'Vist Label', show: true, filter: {
+                    name: 'visitLabel',
+                    type: 'text',
+            }},
+            {label: 'Visit QC Status', show: true, filter: {
+                    name: 'visitQCStatus',
+                    type: 'select',
+                    options: options.visitQCStatus,
+            }},
+            {label: 'First Acquisition', show: true},
+            {label: 'First Insertion', show: true},
+            {label: 'Last QC', show: true},
+            {label: 'New Data', show: true},
+            {label: 'Links', show: true},
+            {label: 'SessionID', show: false},
+            {label: 'Sequence Type', show: false, filter: {
+                name: 'sequenceType',
+                type: 'select',
+                options: options.sequenceTypes,
+            }},
+            {label: 'Pending New', show: false, filter: {
+                name: 'pendingNew',
+                type: 'select',
+                options: options.pendingNew,
+            }},
+        ];
+        Object.values(configLabels).forEach((label)=> {
+            fields.push({label: label + ' QC Status', show: true}
+            );
+        });
+
+        const tabs = [{id: 'browse', label: 'Browse'}];
```

Comment:
Since the imaging browser doesn't use tabs you can remove lines `168`, `172-173`, and `180-181`

Reply:
Removed!